### PR TITLE
05 - Event Planner (Hacktoberfest Demo) #606

### PR DESCRIPTION
**Pull request by Zedoman**
The Event Planner web application allows users to manage events with features like creating, editing, and deleting events. It includes a user-friendly interface displaying event details such as title, description, date, time, location, category, and priority. Events are displayed in card format, making it easy to view and interact with. Additionally, the app integrates Copilot Assistant, which aids users by providing suggestions and automating certain tasks. 

It's built using modern web technologies such as React and integrates a backend API for event management.

Under HacktoberFest issue number #606

### CODE REVIEW

File: community/demos/EventPlanner.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+- Next.js: React Framework
+- React: JavaScript library for building user interfaces
+- CopilotKit: Ai Copilots
+- TypeScript: JavaScript with types
+- TailwindCSS: utility-first CSS framework
+- Shadncn: UI Component Styling
+
+## App Link
+
+- [Snippy Live Demo](https://event-planner-gfex.vercel.app/)
```
Comment (NathanTarbert) [inline community/demos/EventPlanner.md:24]: Name of your app please.

**Comment by NathanTarbert** [comment]
@Zedoman, can you please just add the .md project template file to the demo folder with a link to your repository and remove the project folder?

**Comment by NathanTarbert** [comment]
The app itself looks great, nicely done!

**Comment by Zedoman** [comment]
Sir i have stated the .md file name as EventPlanner.md. and do i have to delete the project folders?

**Comment by Zedoman** [comment]
Sir removed the project folder @NathanTarbert .

File: community/demos/EventPlanner.md (hunk 1 above)
Comment (JeevaRamanathan) [inline community/demos/EventPlanner.md:24]: This can be changed into [Event Planner Demo]

**Review by Zedoman — COMMENTED** [review]
Removed the project folder

**Review by NathanTarbert — APPROVED** [review]

File: community/demos/EventPlanner.md (hunk 1 above)
Comment (Zedoman) [reply community/demos/EventPlanner.md:24]: Event-Planner

**Comment by Zedoman** [comment]
@quest-bot loot #606

**Review by NathanTarbert — APPROVED** [review]

**Comment by NathanTarbert** [comment]
Nice job @Zedoman!

**Comment by Zedoman** [comment]
Thank you sir @NathanTarbert

**Comment by Zedoman** [comment]
@quest-bot loot #636

**Comment by Zedoman** [comment]
@quest-bot loot #695

**Comment by Zedoman** [comment]
@quest-bot embark

**Comment by Zedoman** [comment]
@quest-bot loot #606

**Pull request merged**